src/Dapr.Extensions.Configuration/DaprConfigurationStoreProvider.cs

```diff
@@ -0,0 +1,99 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Dapr.Client;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Primitives;
+
+namespace Dapr.Extensions.Configuration
+{
+    /// <summary>
+    /// A configuration provider that utilizes the Dapr Configuration API. It can either be a single, constant
+    /// call or a streaming call.
+    /// </summary>
+    [Obsolete]
+    public class DaprConfigurationStoreProvider : ConfigurationProvider
+    {
+        private string store { get; }
+        private IReadOnlyList<string> keys { get; }
+        private DaprClient daprClient { get; }
+        private TimeSpan sidecarWaitTimeout { get; }
+        private bool isStreaming { get; } = false;
```

Comment:
By convention properties are `PascalCase`.

Reply:
Nothing needs to access these so I just removed the `{ get; }` and left them camel case.